test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandlerTests.cs

```diff
@@ -118,6 +121,7 @@ public static TheoryData<CreateTokenTheoryData> CreateJWETheoryData
                     new CreateTokenTheoryData
                     {
                         First = true,
+                        IsValid = true,
```

Comment:
If the default value of CreateTokenTheoryData.IsValid was true, you could remove a lot of these sets.